controllers/flinkcluster_converter.go

```diff
@@ -1088,9 +1087,16 @@ log4j.logger.org.apache.flink.shaded.akka.org.jboss.netty.channel.DefaultChannel
         <appender-ref ref="console"/>
     </logger>
 </configuration>`
+)
 
+// TODO: Wouldn't it be better to create a file, put it in an operator image, and read from them?.
+// Provide logging profiles
+func getLogConf(spec v1beta1.FlinkClusterSpec) map[string]string {
+	if spec.LogConfig != nil && spec.LogConfig["log4j-console.properties"] != "" && spec.LogConfig["logback-console.xml"] != "" {
```

Comment:
What if only one of the items is empty? We need to use the default just for that item, right?

Reply:
Certainly can do that, I felt like usually it is all or nothing, and only specifying a single key is indicative of a typo/mistake. But I agree your suggestion is more general/flexible so I will change it.